qlty-check/src/planner.rs

```diff
@@ -268,12 +271,53 @@ impl Planner {
         self.transformers
             .push(Box::new(IssueMuter::new(self.staging_area.clone())));
 
-        // keep overrides last
-        for issue_override in &self.config.overrides {
-            self.transformers.push(Box::new(issue_override.clone()));
+        // keep triage last
+        let triages = self.build_triages();
+        for issue_triage in &triages {
+            self.transformers.push(Box::new(issue_triage.clone()));
         }
     }
 
+    fn build_triages(&self) -> Vec<Triage> {
+        let mut triages = self.config.triage.clone();
+
+        if !self.config.overrides.is_empty() {
+            eprintln!(
+                "{} The `{}` field in qlty.toml is deprecated. Please use `{}` instead.",
+                style("WARNING:").bold().yellow(),
+                style("override").bold(),
+                style("triage").bold()
+            );
+
+            for issue_override in &self.config.overrides {
+                triages.push(Triage {
+                    set: Set {
+                        level: issue_override.level.as_ref().map(|l| level_from_str(l)),
+                        category: issue_override
+                            .category
+                            .as_ref()
+                            .map(|c| category_from_str(c)),
+                        mode: issue_override.mode.map(|m| match m {
+                            IssueMode::Block => PluginMode::Block,
+                            IssueMode::Comment => PluginMode::Comment,
+                            IssueMode::Monitor => PluginMode::Monitor,
+                            IssueMode::Disabled => PluginMode::Monitor, // Mapping Disabled to Monitor as closest equivalent
```

Comment:
Instead of doing this here, I think you can implement the `From` trait to tell Rust how to transform one enum to the other, which will clean up the code